Lib/traceback.py

```diff
@@ -169,7 +169,7 @@ def _some_str(value):
     try:
         return str(value)
     except:
-        return '<unprintable %s object>' % type(value).__name__
+        return '<exception str() failed>'
```

Comment:
The former message contains more information. Would not be better to change other places?

Reply:
The information is duplicated because the exception is printed separately.